navigator/backend/src/main/scala/com/digitalasset/navigator/json/DamlLfCodec.scala

```diff
@@ -91,9 +92,7 @@ object DamlLfCodec {
     case Model.DamlLfPrimType.Timestamp => JsString(tagTypeTimestamp)
     case Model.DamlLfPrimType.Optional => JsString(tagTypeOptional)
     case Model.DamlLfPrimType.Map => JsString(tagTypeMap)
```

Comment:
I assume this can't be renamed to `TextMap` for backwards compatibility reasons?

Reply:
In fact I would like to rename it as well.